lib/gooddata/models/data_source.rb

```diff
@@ -0,0 +1,66 @@
+# encoding: UTF-8
+# frozen_string_literal: true
+#
+# Copyright (c) 2010-2017 GoodData Corporation. All rights reserved.
```

Comment:
Please update the copyright to "# Copyright (c) 2010-2020 GoodData Corporation. All rights reserved."